static/js/stats/stats.js

```diff
@@ -55,6 +56,21 @@ function format_date(date, include_hour) {
     return month_str + ' ' + day + ', ' + year;
 }
 
+function update_last_full_update(end_times) {
+    if (end_times.length === 0) {
+        return;
+    }
+
+    last_full_update = Math.min(last_full_update, end_times[end_times.length - 1]);
+    last_full_update = new Date(last_full_update * 1000);
```

Comment:
As written, this will set the last_full_update to the end_times[end_times.length-1] of the last $.get that returns, not the minimum of all of them. Do you see why?